modules/programs/zsh.nix

```
@@ -392,6 +400,8 @@ in
       home.file."${relToDotDir ".zshrc"}".text = ''
         typeset -U path cdpath fpath manpath
 
+        cdpath+=(${concatStringsSep " " cfg.cdpath})
```

I guess we should add this line only if `cdpath` was specified? Or is it safe to leave it as `cdpath+=()` by default?

```suggestion
        ${optionalString (cfg.cdpath != [])
          cdpath+=(${concatStringsSep " " cfg.cdpath})
        }
```

---

I'm unsure whether it's safe or not, but I'd rather just avoid the issue so thanks for the patch.

---

There was a slight issue with that patch, so I incorporated it manually.